src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue9794.cs

```diff
@@ -15,20 +25,19 @@ public Issue9794(TestDevice testDevice) : base(testDevice)
 		
 		[Test]
 		[Category(UITestCategories.Shell)]
-		[Category(UITestCategories.Compatibility)]
 		public void EnsureTabBarStaysVisibleAfterPoppingPage()
```

Comment:
This test is failing on Android:
<img width="1143" alt="image" src="https://github.com/user-attachments/assets/02979086-387a-4149-b457-9c41e74be05f">